lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -1,5 +1,10 @@
 if Enum.all?(
-     [Membrane.H264.FFmpeg.Parser, Membrane.HTTPAdaptiveStream.SinkBin, Membrane.AAC.FDK.Encoder],
+     [
+       Membrane.H264.FFmpeg.Parser,
+       Membrane.HTTPAdaptiveStream.SinkBin,
+       Membrane.AAC.FDK.Encoder,
+       Membrane.AAC.Parser
+     ],
      &Code.ensure_loaded?/1
    ) do
```

Comment:
I think if we are doing something like this, at the very least it should be mentioned in moduledoc which plugins are required and what you need to do to get them - optional dependencies aren't exactly intuitive.